ppl/src/main/antlr/OpenSearchPPLParser.g4

```diff
@@ -273,6 +275,19 @@ tableSource
     | ID_DATE_SUFFIX
     ;
 
+tableFunction
+    : qualifiedName LT_PRTHS
+        (tableFunctionArgument (COMMA tableFunctionArgument)*)? RT_PRTHS
```

Comment:
```suggestion
    : qualifiedName LT_PRTHS
        (tableFunctionNamedArgument (COMMA tableFunctionNamedArgument )*)? RT_PRTHS
    | qualifiedName LT_PRTHS
        (tableFunctionUnnamedArgument (COMMA tableFunctionUnnamedArgument)*)? RT_PRTHS
```
You can adjust this rule like this, so parser will reject a mix of named and unnamed arguments, e.g.: `source=prometheus.query_range("q", step=2, 42, 15)`